content/docs/get-started/index.md

```diff
@@ -4,97 +4,214 @@ description: 'Learn how you can use MLEM to easily manage and deploy models'
 
 # Get Started
 
-Assuming MLEM is already [installed](/doc/install) in your active python
-environment, let's initialize it by running `mlem init` inside a Git project:
+We assume MLEM is already [installed](/doc/install) in your active Python
+environment, as well as `pandas` and `sklearn`. If not, you can follow the
+instructions below:
 
 <details>
 
 ### ⚙️ Expand for setup instructions
 
-If you want to follow along with this tutorial and try MLEM, you can use our
-[example repo](https://github.com/iterative/example-mlem-get-started). You'll
-need to [fork] it first (so you can push models). Then clone it locally:
-
-[fork]: https://docs.github.com/en/get-started/quickstart/fork-a-repo
-
-```cli
-$ git clone <your fork>
-$ cd example-mlem-get-started
-```
-
-Next let's create an isolated virtual environment to cleanly install all the
-requirements (including MLEM) there:
+Let's create a separate folder and an isolated virtual environment to cleanly
+install all the requirements (including MLEM) there:
 
 ```cli
+$ mkdir mlem-get-started
+$ cd mlem-get-started
 $ python3 -m venv .venv
 $ source .venv/bin/activate
```

Comment:
Linux/Mac-specific.

Do we expect Windows users? Maybe use tabs if so ([example](https://dvc.org/doc/start/data-management/data-versioning#expand-to-set-up-remote-storage)).

Same for other commands (e.g. `tree` and pages in the GS).

Reply:
as per analytics, only 3% of MLEM command calls are from windows. So let's skip it for now